src/api-guides/travel/deeplink-integration-new.md

```diff
@@ -203,12 +203,59 @@ https://www.concursolutions.com/goto/hotel-shop?checkindate=2024-11-02&checkoutd
+
+
+#### URI Template
+
+```
+https://www.concursolutions.com/goto/hotel-shop?tripid={tripid}&location={location}&locationsource={locationsource}&distance={distance}&unit={unit}
+```
+
+#### Example of Usage
+```
+https://www.concursolutions.com/goto/hotel-shop?tripid=9be19dda-bf33-4110-ac48-bf840878cd3c&location=50.050686,8.565997&distance=15&unit=mi
+```
+
+#### Query Parameters
+
+|Name            |Type     |Format                                                                                                                                             |Description|
+|----------------|---------|---------------------------------------------------------------------------------------------------------------------------------------------------|-----------|
+|`tripid`        |`string` |`UUID`                                                                                                                                             |**Required** <br>Unique identifier of the trip, formatted as a UUID.|
+|`location`      |`string` |[Location Format](/api-guides/travel/deeplink-integration.html#location-format-hotel)                                                              |(Optional) <br>Location of the search center with default search radius of 5 mi.|
+|`locationsource`|`string` |one of {`northstar`, `giata`, `leonardo`, `amadeus`, `sabre`, `galileo`, `cwt`, `expedia`, `hrs`, `booking.com`, `omnibees`}                       |**Required if location is provided and is a Hotel Property ID**. <br>Source of the Hotel Property ID. For geo-coordinates and SAP Concur Travel Internal Hotel ID, omit this query parameter.|
+|`distance`      |`integer`|one of {`1`, `2`, `3`, `4`, `5`, `10`, `15`, `20`, `25`, `30`, `35`, `40`, `45`, `50`, `55`, `60`, `65`, `70`, `75`, `80`, `85`, `90`, `95`, `100`}|(Optional) <br> Distance around the `location`. <br>If not set, it will default to 5 mi `distance` and `radius` as search radius.|
+|`unit`          |`string` |`km` or `mi`                                                                                                                                       |(Optional) <br> Unit for the `distance`. <br>If not set, the value will be taken from users profile - under Profile Settings, System Settings, **mile/km**.|
+
+#### <a name="example-add-to-trip"></a> Example: Only trip ID
+```
+https://www.concursolutions.com/goto/hotel-shop?tripid=a7f172cd-8587-48d5-8533-5ddb7ff5c26c
+```
+
+This deeplink will pre-fill the hotel search form to add a booking to the trip with id `a7f172cd-8587-48d5-8533-5ddb7ff5c26c`. The location, check-in and check-out dates will be derived from the existing bookings of the trip. By default the first booking of the trip will be used, but the user can select another one in the form itself.
```

Comment:
That will avoid us having to adapt the docu in case something with the hotel search proposals will change.
```suggestion
This deeplink will redirect the user to the hotel search form to add a booking to the trip with id `a7f172cd-8587-48d5-8533-5ddb7ff5c26c`. Search proposals based on the content of the trip are available in the search form.
```